src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,47 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.5.0
+14 February, 2023
+
+### Breaking Change
+
+- Create dedicated registry serviceAccounts by default
+
+### Features
+
+- Added antiAffinity (configuration)[/docs/self-hosted/administration/configuration/#userpodaffinity] for PODs with label `dev.okteto.com/affinity`
+- Add external resource icons
+- Build credential helpers
+- Remove load time on GitHub redeploy dialog
+
+### Bug Fix
+
+- When injecting host alises to go through internal network to buildkit, registry and api, don't duplicate entries for same IP. Instead, pass the list of hostnames for the IP
+- Delete button is now disabled in the personal namespace
+- Restore k3s support
+- Fix faulty error case on deployment with external resource
+- Fix faulty progressing state on deployment with external resource
+- Disable CTA while destroying all
```

Comment:
What do we mean by CTA here? The banner?

Reply:
@tavferreira https://github.com/okteto/app/pull/5453 can you give more contexT?

Reply:
Sorry, I should have been more descriptive here. I'll keep it in mind in the future.

It refers to the `Launch dev environment` CTA when a space is empty. 
In some cases, the button was enabled even when the destroying action was still taking action.